apis/traffic-access/v1alpha4/traffic-access.md

```diff
@@ -0,0 +1,280 @@
+# Traffic Access Control
+
+**API Group:** access.smi-spec.io
+
+**API Version:** v1alpha4
```

Comment:
The API version should be v1alpha1-draft

Reply:
You mean `v1alpha4-draft`? I've changed it to `v1alpha4-draft` instead.

![image](https://user-images.githubusercontent.com/13926417/133343179-03b275f1-07e2-4e47-955e-27179df5c56a.png)